### PR TITLE
article/salesforce-vs-custom-development

### PR DESCRIPTION
**Pull request by sapanparikh18**
Salesforce vs customer development

### CODE REVIEW

File: content/blog/salesforce-vs-in-house-custom-development.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+## The Impact
+
+Genie started saving 4000 hours a day across 800 users.
+
+## What Incubyte Achieved
+
+- **Saved 4000 hours a day.** Team members onboarded by the Metamorphosis project have a seamless workflow that resulted in time saving
+- **Over 800 users onboarded** by maintaining smaller, stable, consistent releases
+- **It's a tailored solution**, it is more than customization, because Incubyte developed a solution with the end user and their use cases in mind.
+- **By being fast, economical, and agile**, within two years, Metamorphosis added value to the company
```
Comment (sajagjain) [inline content/blog/salesforce-vs-in-house-custom-development.md:88]: This line does not sound impactful. Can we change it to something like, in a short span of 2 yrs, Metamorphis was able to add value to the company?

File: content/blog/salesforce-vs-in-house-custom-development.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+| Operational CTC for 100 users = 270,000$ | Operational CTC per year for 800+ users = 100,000$ |
+| Typical Lead Time to Release = Undefined | Typical Lead Time to Release = 2 Weeks |
+
+## The Impact
+
+Genie started saving 4000 hours a day across 800 users.
+
+## What Incubyte Achieved
+
+- **Saved 4000 hours a day.** Team members onboarded by the Metamorphosis project have a seamless workflow that resulted in time saving
```
Comment (sajagjain) [inline content/blog/salesforce-vs-in-house-custom-development.md:85]: 4000 hours a day does not sound right. Do we mean 4000 hours a week maybe?

File: content/blog/salesforce-vs-in-house-custom-development.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+- **Over 800 users onboarded** by maintaining smaller, stable, consistent releases
+- **It's a tailored solution**, it is more than customization, because Incubyte developed a solution with the end user and their use cases in mind.
```
Comment (sajagjain) [inline content/blog/salesforce-vs-in-house-custom-development.md:87]: Can we change their use case in mind to the customer's use case in mind?

File: content/blog/salesforce-vs-in-house-custom-development.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+A low-code CRM can be very powerful once built and launched well. They are ideal for running campaigns, marketing outreach, and more.
+
+CRMs like Salesforce would work well in a business model where the nature of things is very well defined but the moment multiple systems are involved with different data points, its power weakens. In a nuanced environment, with unique use cases and intricacies, it is just as hard to work with something like Salesforce as it is to develop a tailored software solution to your specifications.
+
+When it comes to adopting solutions like Salesforce, a company would need to pay for the portal (license) and pay separately for the additional development services the business needs. Whereas custom software shifts the cost from Op-ex to Cap-ex.
+
+Consider your requirements and choose wisely. Will your end users absolutely benefit from the CRM or could they do more with a custom tool?
+
+## The power of Metamorphosis, a.k.a. Custom Tailored Development
+- **Ecosystem built per your need** so keeps your business, client and employees in mind from day one.
```
Comment (sajagjain) [inline content/blog/salesforce-vs-in-house-custom-development.md:104]: Maybe change it to so keeps **the needs of** your business, client...

File: content/blog/salesforce-vs-in-house-custom-development.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+- **Cost shifts from opex to capex** once built you do not have to pay additional per user you add to the ecosystem
```
Comment (sajagjain) [inline content/blog/salesforce-vs-in-house-custom-development.md:105]: Maybe change it to, once built there is no need for additional expenditure to onboard new users. You can choose to ignore this if you want to include per user keyword

File: content/blog/salesforce-vs-in-house-custom-development.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+- **Can be faster cheaper and better** if done right
+- **With Incubyte**, your software can be reliable, scalable, and significantly affordable
+
+Choosing a direction to tackle a system overhaul is never easy. There are no simple decisions, and every action has long-lasting consequences. We hope this study helped you understand your options better and what they ultimately translate to for your business. Contact us if you want to know how tailored digital products can uplift your business and enable your employees. We would love to help you with your Metamorphosis.
```
Comment (sajagjain) [inline content/blog/salesforce-vs-in-house-custom-development.md:109]: Do you think the last line needs to be a newline with maybe a more emphasized font size?

File: content/blog/salesforce-vs-in-house-custom-development.md (hunk 2 above)
Comment (sapanparikh18) [reply content/blog/salesforce-vs-in-house-custom-development.md:85]: Some of these numbers have come from Padma, @sajagjain  please get Padma and Prathima on call and let's wrap this up

**Review by sajagjain — APPROVED** [review]

File: content/blog/salesforce-vs-in-house-custom-development.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+| Typical Lead Time to Release = Undefined | Typical Lead Time to Release = 3 Weeks             |
+
+## The Impact
+
+Genie started saving 1000 hours a day across 225+ daily active users.
+
+## What Incubyte Achieved
+
+- **Saved 4000 hours a day.** Team members onboarded by the Metamorphosis project have a seamless workflow that resulted in time saving
+- **Over 800 users onboarded** by maintaining smaller, stable, consistent releases
```
Comment (sajagjain) [inline content/blog/salesforce-vs-in-house-custom-development.md:86]: Should this be 900?

**Pull request merged**